src/components/App/App.js

```diff
@@ -0,0 +1,28 @@
+require('normalize.css/normalize.css')
+require('./App.css')
+
+import React from 'react'
+import Navbar from '../Navbar/Navbar'
+import { Switch, Route } from 'react-router-dom'
+import Shop from '../../screens/Shop/Shop'
+import ShoppingCart from '../../screens/ShoppingCart/ShoppingCart'
+import ProductDetails from '../../screens/ProductDetails/ProductDetails'
```

Comment:
Relative paths can be hard to follow / maintain in larger projects. We can use absolute paths (relative to project `src`) adding `webpack` config something like

`resolve: { modules: [path.resolve('./src'), etc..]}`
  
then you could write 
`import ProductDetails from 'screens/ProductDetails/ProductDetails'`